test-suite-kotlin/src/test/kotlin/io/micronaut/docs/server/intro/HelloController.kt

```diff
@@ -15,13 +15,11 @@
  */
 package io.micronaut.docs.server.intro
 
-import io.micronaut.context.annotation.Requires
-
 // tag::imports[]
+import io.micronaut.context.annotation.Requires
```

Comment:
The requires annotation was intentionally left out of the imports because its not visible in the class tag

Reply:
moved it back, my bad there